Manuscript/Text/02F_mutein.md

```diff
@@ -1,3 +1,9 @@
+
+It is desirable for many clinical applications for a drug to induce a purely immunosuppresive response. In order to quantify the effectiveness of these modified and wild-type cytokines in selectively activating immunosuppressive T~reg~ cells, we defined formulated specificity functions. These functions were defined as pSTAT5 activity of T~reg~s divided by pSTAT5 activity of T-helper cells or NK cells. Specificty of T~reg~ activation with respect to NK and T-helper cells was experimentally determined and predicted by our model to be largest at low concentrations of IL-2, but was not found to have a significant relationship with IL-15 concentration ([@fig:mutFac]B/C). Because high T~reg~ specificity is desirable for several clinical applications of cytokine-based drugs, we also quantified the partial derivatives of our specificity functions with respect to both surface and endosomal binding constants, and found that increasing IL-2Rα affinity while decreasing IL-2Rβ/γ~c~ affinity on the cells surface would most drastically increase the cytokine's T~reg~ activation specificity ([@fig:mutFac]D). These effects were notably different between surface and endosomal binding rates. Furthermore, our model predicts that ligands with reduced IL-2Rα affinity had a decreased ability to specifically activate T~reg~s with respect to NK and T-Helper cells over any degree of IL-2Rβ/γ~c~ affinity reduction ([@fig:mutFac]E). (put something specifically about current industry drug-discovery/design here?) Our model was also able to accurately guage the effect that changes in ligand-receptor binding affinities had on their respective EC~50~s ([@fig:mutFac]F). Ligands with decreased IL-2Rα affinity had increased EC~50~s for T~reg~ varieties, and ligands with decreased IL-2Rβ/γ~c~ affinities had larger EC~50~s for T-Helper cells, in accordance with our previous findings.
+
+In order to better visualize the experimental results when PBMCs were exposed to our muteins and modified IL-2 ligands, we performed tensor factorization on a tensor containing both wild-type and modified IL-2 ligands experimental results ([@Fig:mutFac]G-J). Using a tensor factorization with two components explained roughly 80% of the variance in the new combined data tensor. Among the ligands, wild-type IL-2/15 and wilde-type IL-2 conjugated to an Fc domain at it's N-terminus are most correlated to T~reg~s via their shared correlation with component 2 ([@Fig:mutFac]G-J). This suggests that cytokines conjugated to domains at their N-terminus are more strongly suited to induce selective T~reg~ activation than those conjugated at their C-terminus, which is likely due to these conjugation types' opposing effects on the cytokine's IL-2Rα affinity.
```

Comment:
Maybe something along the lines of: "Like in Fig. 3, visualizing the effect of changing a cytokine treatment is complicated by the effects spanning distinct cells, changing with concentration, and varying over time..."

Reply:
Also, add a last summary sentence. E.g. "In total, our results demonstrate our model works, and identify new strategies for further engineering cell specificity... "